### PR TITLE
Add developer velocity use case to HB

### PR DESCRIPTION
**Pull request by dan-mckean**
From [this doc](https://docs.google.com/document/d/19Ijpe6Jjxedr3qudkzhSMArDTi-oU4r6UIVGQhMTRm4/edit#heading=h.urii3fcnn0k4).

### CODE REVIEW

File: content/strategy-goals/strategy/index.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Our [values](../../company-info-and-process/values/index.md) are the principles
 
 We have identified five key use cases for our universal code search platform:
 
-1. Developer onboarding & velocity
+1. [Developer onboarding & velocity](use_case/dev_velocity.md)
```
Comment (dan-mckean) [inline content/strategy-goals/strategy/index.md]:
```suggestion
1. [Developer onboarding & velocity](use_cases/dev_velocity.md)
```

File: content/strategy-goals/strategy/use_cases/dev_velocity.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# Developer onboarding & velocity use case
+
+## Why
+
```
Comment (jasonyavorsky) [inline content/strategy-goals/strategy/use_cases/dev_velocity.md]:
```suggestion

<!-- Beyond imagining a future in the above section, talk more about why this future is important and why we are going after it. -->
```

File: content/strategy-goals/strategy/use_cases/dev_velocity.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+1. Knowing where the code is or being able to find it
+1. Knowing what dependencies there are on the code (i.e. what would be the impact of changing the code)
+1. Knowing what dependencies the code depends on (i.e. internal/external libraries used)
+1. Knowing who is responsible for or has contextual knowledge about the code
+1. Knowing who updated the code and why
+
+This kind of understanding is central to both onboarding and velocity, and research shows that revenue growth among companies who invest in the velocity of their developers see revenue growth upto 4-5 times faster than those who don’t.
+
+## Vision
+
```
Comment (jasonyavorsky) [inline content/strategy-goals/strategy/use_cases/dev_velocity.md]:
```suggestion

<!-- Convey what things will be like in the future for your use case, being as descriptive as you can to help someone understand where we are headed with our vision. -->
```

File: content/strategy-goals/strategy/use_cases/dev_velocity.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+Sourcegraph anchors onboarding inside engineering organizations and enables engineers to dive into unfamiliar code and get to "first bug fix or feature" quickly. With Soucegraph integrated in the developers workflow they see increased velocity in their work over their peers that do not integrate Sourcegraph in their workflow.
+
+## How we solve this today
+
```
Comment (jasonyavorsky) [inline content/strategy-goals/strategy/use_cases/dev_velocity.md]:
```suggestion

<!-- Describe in as much detail as you can how the product enables this use case today. You can include customer quotes, textual walkthroughs, and this is also a great place to link to demo videos. This is perhaps the most important single section in this document, so don't be afraid to add too much - if you feel this section is getting long, consider summarizing here and linking out to other pages in the handbook with details. -->
```

File: content/strategy-goals/strategy/use_cases/dev_velocity.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+Despite the lack of deliberate focus on solving this problem, the following are real, anonymized quotes from customers today on how we’re already helping them with this use case:
+
+- Sourcegraph is our answer for sharing information and facilitating easy collaboration across teams, despite the boundaries of distance and time
+- There’s all these little hidden things that tax the velocity of the teams. Earlier this month I was using Sourcegraph to find code and things we needed to replace, related to updating our build system, and we’ve seen an improvement in the build times as a result. But I feel like the summation of all these little things actually adds up to way more than just speeding up the build time.
+- Sourcegraph increases the efficiency and confidence of our entry level developers when they build features that touch different parts of our code base.
+- For our new developers, Sourcegraph has been invaluable to get to know the repository structure, to track down where code lives, and self-service during their investigations.
+
+## Who benefits
+
```
Comment (jasonyavorsky) [inline content/strategy-goals/strategy/use_cases/dev_velocity.md]:
```suggestion

<!-- Link to the personas that relate to this use case, and describe briefly how it benefits each of them (the real detail is in the above section, so be sure not to repeat yourself here; speak in generalities for each persona in this section.) -->
```

File: content/strategy-goals/strategy/use_cases/dev_velocity.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+[Engineering Leader](https://docs.google.com/presentation/d/1aQhcWoWd_LJXdAgEn7JBGnZV5pfN6UJyct2VV-ZiTXI/edit#slide=id.ge9b93ff711_0_19)
+
+- Boost your teams’ experience and productivity
+- Create a best-in-class onboarding experience for your new hires
+
+[Dev Productivity Lead](https://docs.google.com/presentation/d/1aQhcWoWd_LJXdAgEn7JBGnZV5pfN6UJyct2VV-ZiTXI/edit#slide=id.ge9b93ff711_0_19)
+
+- Allow your engineers to move faster
+- Eliminate single point of failure and knowledge silos: every dev can self-serve and troubleshoot
+- Help surface best practices across the team
```
Comment (jasonyavorsky) [inline content/strategy-goals/strategy/use_cases/dev_velocity.md]:
```suggestion
- Help surface best practices across the team

## Additional resources

<!-- Are there other articles, blogs, internal documents, or handbook links that are useful for someone who wants to understand this use case? Link to them here. -->

- [Use case index](../index.md#use-cases)
-- [Productboard developer onboarding & velocity (internal only)](https://sourcegraph.productboard.com/feature-board/3957049-fy23-use-cases/features/11482294/detail)
```

File: content/strategy-goals/strategy/use_cases/dev_velocity.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+1. Being able to form a clear mental model of the overall system (i.e. knowing the various parts and how they relate)
+1. Knowing where the code is or being able to find it
+1. Knowing what dependencies there are on the code (i.e. what would be the impact of changing the code)
+1. Knowing what dependencies the code depends on (i.e. internal/external libraries used)
+1. Knowing who is responsible for or has contextual knowledge about the code
+1. Knowing who updated the code and why
+
+This kind of understanding is central to both onboarding and velocity, and research shows that revenue growth among companies who invest in the velocity of their developers see revenue growth upto 4-5 times faster than those who don’t.
+
+## Vision
```
Comment (jasonyavorsky) [inline content/strategy-goals/strategy/use_cases/dev_velocity.md]:
```suggestion
## Overall vision
```

File: content/strategy-goals/strategy/use_cases/dev_velocity.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
+# Developer onboarding & velocity use case
+
+## Why
```
Comment (jasonyavorsky) [inline content/strategy-goals/strategy/use_cases/dev_velocity.md]:
```suggestion
## Why this is important
```

File: content/strategy-goals/strategy/use_cases/dev_velocity.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
+# Developer onboarding & velocity use case
+
```
Comment (jasonyavorsky) [inline content/strategy-goals/strategy/use_cases/dev_velocity.md]:
```suggestion

This page provides a strategic introduction to one of our [company use cases](../index.md#use-cases). Check that link to find the rest of the use cases and learn how we use them as part of our company strategy.

```

**Comment by jasonyavorsky** [comment]
I made a few suggestions so it matches all the other use case documents in terms of comments, boilerplate, heading names, etc. The order of headings is different here and I didn't make a suggestion on that in case it was on purpose.

File: content/strategy-goals/strategy/index.md [hunk 10]
```diff
@@ -29,7 +29,7 @@ Our [values](../../company-info-and-process/values/index.md) are the principles
 
 We have identified five key use cases for our universal code search platform:
 
-1. Developer onboarding & velocity
+1. [Developer onboarding & velocity](use_cases/dev_velocity.md)
```
Comment (jasonyavorsky) [inline content/strategy-goals/strategy/index.md]: Maybe include onboarding in URL for better search results? Not sure if it matters.

Comment (jeanduplessis) [reply content/strategy-goals/strategy/index.md]:
```suggestion
1. [Developer onboarding & velocity](use_cases/dev-onboarding.md)
```

File: content/strategy-goals/strategy/use_cases/dev-onboarding.md [hunk 11]
```diff
@@ -0,0 +1,55 @@
+# Developer onboarding & velocity use case
```
Comment (jeanduplessis) [inline content/strategy-goals/strategy/use_cases/dev-onboarding.md]:
```suggestion
# Developer onboarding
```

File: content/strategy-goals/strategy/index.md [hunk 12]
```diff
@@ -29,7 +29,7 @@ Our [values](../../company-info-and-process/values/index.md) are the principles
 
 We have identified five key use cases for our universal code search platform:
 
-1. Developer onboarding & velocity
+1. [Developer onboarding & velocity](use_cases/dev-onboarding.md)
```
Comment (jeanduplessis) [inline content/strategy-goals/strategy/index.md]:
```suggestion
1. [Developer onboarding](use_cases/dev-onboarding.md)
```

File: content/strategy-goals/strategy/use_cases/dev-onboarding.md [hunk 13]
```diff
@@ -0,0 +1,55 @@
...
+1. Knowing where the code is or being able to find it
+1. Knowing what dependencies there are on the code (i.e. what would be the impact of changing the code)
+1. Knowing what dependencies the code depends on (i.e. internal/external libraries used)
+1. Knowing who is responsible for or has contextual knowledge about the code
+1. Knowing who updated the code and why
+
+This kind of understanding is central to both onboarding and velocity, and research shows that revenue growth among companies who invest in the velocity of their developers see revenue growth upto 4-5 times faster than those who don’t.
+
+## Vision
+
```
Comment (jeanduplessis) [inline content/strategy-goals/strategy/use_cases/dev-onboarding.md:25]:
```suggestion
<!-- Convey what things will be like in the future for your use case, being as descriptive as you can to help someone understand where we are headed with our vision. -->

```

File: content/strategy-goals/strategy/use_cases/dev-onboarding.md [hunk 14]
```diff
@@ -0,0 +1,55 @@
+# Developer onboarding
+
+## Why
```
Comment (jeanduplessis) [inline content/strategy-goals/strategy/use_cases/dev-onboarding.md]:
```suggestion
## Why this is important
<!-- Beyond imagining a future in the above section, talk more about why this future is important and why we are going after it. -->
```

File: content/strategy-goals/strategy/use_cases/dev-onboarding.md [hunk 15]
```diff
@@ -0,0 +1,55 @@
...
+1. Knowing who is responsible for or has contextual knowledge about the code
+1. Knowing who updated the code and why
+
+This kind of understanding is central to both onboarding and velocity, and research shows that revenue growth among companies who invest in the velocity of their developers see revenue growth upto 4-5 times faster than those who don’t.
+
+## Vision
+
+Sourcegraph anchors onboarding inside engineering organizations and enables engineers to dive into unfamiliar code and get to "first bug fix or feature" quickly. With Soucegraph integrated in the developers workflow they see increased velocity in their work over their peers that do not integrate Sourcegraph in their workflow.
+
+## How we solve this today
```
Comment (jeanduplessis) [inline content/strategy-goals/strategy/use_cases/dev-onboarding.md:28]:
```suggestion
## How we solve this today
<!-- Describe in as much detail as you can how the product enables this use case today. You can include customer quotes, textual walkthroughs, and this is also a great place to link to demo videos. This is perhaps the most important single section in this document, so don't be afraid to add too much - if you feel this section is getting long, consider summarizing here and linking out to other pages in the handbook with details. -->
```

File: content/strategy-goals/strategy/use_cases/dev-onboarding.md [hunk 16]
```diff
@@ -0,0 +1,55 @@
...
+
+Despite the lack of deliberate focus on solving this problem, the following are real, anonymized quotes from customers today on how we’re already helping them with this use case:
+
+- Sourcegraph is our answer for sharing information and facilitating easy collaboration across teams, despite the boundaries of distance and time
+- There’s all these little hidden things that tax the velocity of the teams. Earlier this month I was using Sourcegraph to find code and things we needed to replace, related to updating our build system, and we’ve seen an improvement in the build times as a result. But I feel like the summation of all these little things actually adds up to way more than just speeding up the build time.
+- Sourcegraph increases the efficiency and confidence of our entry level developers when they build features that touch different parts of our code base.
+- For our new developers, Sourcegraph has been invaluable to get to know the repository structure, to track down where code lives, and self-service during their investigations.
+
+## Who benefits
```
Comment (jeanduplessis) [inline content/strategy-goals/strategy/use_cases/dev-onboarding.md:45]:
```suggestion
## Who benefits
<!-- Link to the personas that relate to this use case, and describe briefly how it benefits each of them (the real detail is in the above section, so be sure not to repeat yourself here; speak in generalities for each persona in this section.) -->
```

File: content/strategy-goals/strategy/use_cases/dev-onboarding.md [hunk 17]
```diff
@@ -0,0 +1,55 @@
...
+[Engineering Leader](https://docs.google.com/presentation/d/1aQhcWoWd_LJXdAgEn7JBGnZV5pfN6UJyct2VV-ZiTXI/edit#slide=id.ge9b93ff711_0_19)
+
+- Boost your teams’ experience and productivity
+- Create a best-in-class onboarding experience for your new hires
+
+[Dev Productivity Lead](https://docs.google.com/presentation/d/1aQhcWoWd_LJXdAgEn7JBGnZV5pfN6UJyct2VV-ZiTXI/edit#slide=id.ge9b93ff711_0_19)
+
+- Allow your engineers to move faster
+- Eliminate single point of failure and knowledge silos: every dev can self-serve and troubleshoot
+- Help surface best practices across the team
```
Comment (jeanduplessis) [inline content/strategy-goals/strategy/use_cases/dev-onboarding.md:65]:
```suggestion
- Help surface best practices across the team

- Help surface best practices across the team
## Additional resources
<!-- Are there other articles, blogs, internal documents, or handbook links that are useful for someone who wants to understand this use case? Link to them here. -->
- [Use case index](../index.md#use-cases)
-- [Productboard developer onboarding & velocity (internal only)](https://sourcegraph.productboard.com/feature-board/3957049-fy23-use-cases/features/11482294/detail)
```

File: content/strategy-goals/strategy/use_cases/dev-onboarding.md [hunk 18]
```diff
@@ -0,0 +1,55 @@
...
+1. Being able to form a clear mental model of the overall system (i.e. knowing the various parts and how they relate)
+1. Knowing where the code is or being able to find it
+1. Knowing what dependencies there are on the code (i.e. what would be the impact of changing the code)
+1. Knowing what dependencies the code depends on (i.e. internal/external libraries used)
+1. Knowing who is responsible for or has contextual knowledge about the code
+1. Knowing who updated the code and why
+
+This kind of understanding is central to both onboarding and velocity, and research shows that revenue growth among companies who invest in the velocity of their developers see revenue growth upto 4-5 times faster than those who don’t.
+
+## Vision
```
Comment (jeanduplessis) [inline content/strategy-goals/strategy/use_cases/dev-onboarding.md]:
```suggestion
## Overall vision
```

File: content/strategy-goals/strategy/use_cases/dev-onboarding.md [hunk 19]
```diff
@@ -0,0 +1,55 @@
+# Developer onboarding
```
Comment (jeanduplessis) [inline content/strategy-goals/strategy/use_cases/dev-onboarding.md:1]:
```suggestion
# Developer onboarding
This page provides a strategic introduction to one of our [company use cases](../index.md#use-cases). Check that link to find the rest of the use cases and learn how we use them as part of our company strategy.
```

File: content/strategy-goals/strategy/use_cases/dev-onboarding.md [hunk 20]
```diff
@@ -0,0 +1,74 @@
...
+
+## How we solve this today
+
+<!-- Describe in as much detail as you can how the product enables this use case today. You can include customer quotes, textual walkthroughs, and this is also a great place to link to demo videos. This is perhaps the most important single section in this document, so don't be afraid to add too much - if you feel this section is getting long, consider summarizing here and linking out to other pages in the handbook with details. -->
+
+Today we do not specifically focus on solving this problem, however we know from speaking with existing customers they use Sourcegraph to enhance developer onboarding & velocity. Below are two case studies further elaborating these notions.
+
+[Convoy improved on-boarding](https://about.sourcegraph.com/case-studies/convoy-improved-on-boarding/)
+
+[Indeed accelrates developmeent velocity](https://about.sourcegraph.com/case-studies/indeed-accelerates-development-velocity/)
```
Comment (jeanduplessis) [inline content/strategy-goals/strategy/use_cases/dev-onboarding.md]:
```suggestion
[Indeed accelerates development velocity](https://about.sourcegraph.com/case-studies/indeed-accelerates-development-velocity/)
```

**Comment by jeanduplessis** [comment]
I'm taking a bias to action and merging it as is. We can keep iterating on still.

As per the conversations in Slack, I've removed the "developer velocity" part of the use case title. We still speak to velocity in the content as it's a key outcome of effective onboarding.

**Pull request merged**